src/repositories/schemas.ts

```diff
@@ -1,9 +1,195 @@
 import { OAuth1Client, ObjectId } from '@extrahorizon/javascript-sdk';
 
+interface Transition {
```

Comment:
Is there a reason we chose not to import this from the SDK? 🤔

Reply:
For this PR: just copied from the dataservice file above as well 😄 Will add it to the ticket for cleaning up this file as well https://github.com/ExtraHorizon/exh-cli/pull/119#discussion_r1467479856